examples/javascript/script/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Intentional that you added this file?

Reply:
yeah, why not? since yarn.lock is supposed to be committed, no?

Reply:
Possible, I just noticed that all the other things were changes, and this the only addition.